src/Navigation.js

```diff
@@ -10,11 +12,16 @@ class Navigation extends Component {
 
     return (
       <Navbar bg="light" expand="lg">
-        <Navbar.Brand>Rcruitme</Navbar.Brand>
+        <Navbar.Brand>
+          <Link to='/' >Rcruitme</Link>
+        </Navbar.Brand>
         <Nav className="mr-auto">
         </Nav>
+
+
         <Nav>
-          {auth.loggedIn() ? <SignOut /> : <CreateAccount />}
+          { !auth.loggedIn() && <CreateAccount /> }
```

Comment:
Just a note: may potentially have to rethink how we're handling / keeping track of logged in state at some point. Currently, changes not accounted for in render lifecycle.

Reply:
As in since we're using local storage if that changes, that would not necessarily cause this component to reupdate?

Reply:
Ya